protos/sources/vega/markets.proto

```diff
@@ -245,11 +245,11 @@ message LiquidityMonitoringParameters {
 }
 
 message LiquiditySLAParameters {
+  reserved 3; // Deprecated "providers_fee_calculation_time_step"
```

Comment:
Should this be deprecated rather than removed? If yes deprecated, then could you also deprecate in GQL?